app/football_now.rb

```diff
@@ -2,88 +2,90 @@
 
 require 'httparty'
 require 'active_support/core_ext/string/conversions'
+require 'active_support/isolated_execution_state'
 require 'paint'
 
 # TODO: add docker_build.sh and docker_run.sh scripts to build and run the docker's commands
 
 class FootballNow
-    class << self
-        BASE_URL = 'https://www.zerozero.pt/rss/zapping.php'
-        
-        def start            
-            response = HTTParty.get(BASE_URL)
-            games = response.parsed_response['rss']['channel']['item']
+  class << self
+    BASE_URL = 'https://www.zerozero.pt/rss/zapping.php'
 
-            city_prompt
+    def start
+      response = HTTParty.get(BASE_URL)
+      games = response.parsed_response['rss']['channel']['item']
 
-            print_games(games)
+      city_prompt
 
-        rescue SocketError => e
-            puts "#{Paint['Check your internet connection. Error message:', :red]} #{e.message}"
-        end
+      print_games(games)
+    rescue SocketError => e
+      puts "#{Paint['Check your internet connection. Error message:', :red]} #{e.message}"
+    end
 
-        private
+    private
 
-        def city_prompt
-            puts "\nPORTUGUESE SPORT TV CHANNELS"
-            puts "\nEnter the city to convert games date/times into"
-            puts "\nNOTE: type #{Paint['help', :green]} for the list of cities/timezones\n"
+    def city_prompt
+      puts "\nPORTUGUESE SPORT TV CHANNELS\n\n"\
+        "Enter the city to convert games date/times into\n"\
+        "** type #{Paint['help', :green]} for the list of cities/timezones **\n".center(10)\
```

Comment:
Would this also work?

```ruby
content <<-TEXT
  PORTUGUESE SPORT TV CHANNELS

  Enter the city to convert games date/times into
  ** type #{Paint['help', :green]} for the list of cities/timezones **\n
TEXT

puts content.center(10)
```

If applicable, this is ,ore readable for me. Less escape characters, less distraction.

Reply:
In Ruby, you could also wrap your multiline text inside back ticks (`), I believe. Not sure, though.

Nevertheless, since the HEREDOC syntax is well known among web devs (thanks to php), I would stick to it.

Reply:
I agree, it was in my mental todo list, might as well do it now indeed